examples/fleet-b.yaml

```diff
@@ -30,16 +30,10 @@ spec:
         - name: motd-update
           configType: InlineConfigProviderSpec
           inline:
-            ignition:
-              version: 3.4.0
-            storage:
-              files:
-                - contents:
-                    source: >-
-                      data:,This%20system%20is%20managed%20by%20flightctl.%0A
-                  mode: 422
-                  overwrite: true
-                  path: "/etc/motd"
+            - path: "/etc/motd"
```

Comment:
Could we also update the examples in the flightct-demos repo?
Since it's a breaking change, those examples won't work. (The UI won't either, but we will start working on it soon)

Reply:
Yes of course